### PR TITLE
Feat phone and query fixes

### PR DESCRIPTION
**Pull request by jorgenbele**
Improve incidentsview (use query params, ui improvements), add phone number selector to profiles view, fix acks and more

### CODE REVIEW

**Comment by jorgenbele** [comment]
This closes #107 by making it possible to set a phone number for all notification profiles.

**Review by hmpf — APPROVED** [review]
Hand-tested. This should have been three PRs but time is of the essence so I get it.

Nitpick: If you change something in a notificationprofile, the button says "Create", not "Update". Not ideal UX, but it can wait a while longer.

Also not important now: websockets is acting up and spewing warnings:
```2020-09-15 09:26:00,467 daphne.ws_protocol ERROR    [Failure instance: Traceback: <class 'ValueError'>: No route found for path 'ws/active/'.```
Needs to be looked at for the final polish, though.

I'll approve this regardless, but: it is not possible to create a filter in the filter-view that filters on open/acked/stateless. We'll need that for production, but not the demo.

File: src/api/index.tsx [hunk 1]
```diff
@@ -198,24 +212,14 @@ export interface Acknowledgement {
...
-
-export type PhoneNumberRequest = Omit<PhoneNumber, "pk" | "user">;
-export type PhoneNumberSuccessResponse = PhoneNumber;
-
-export interface PhoneNumber {
-  pk: number;
-  user: number;
-  phone_number: string;
-}
-
```
[the remark this answers is not in the document]
Comment (hmpf) [reply src/api/index.tsx:218]: Adding a phone number still works though.

[the remark this answers is not in the document]
Comment (jorgenbele) [reply src/api/index.tsx:218]: When I added the PhoneNumber to the NotificationProfile type I had to move it for it to be declared before use.

**Pull request merged**